packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -420,6 +420,8 @@ module.exports.subscriptionsCustomerValidator = isA.object({
   payment_type: isA.string().optional(),
   paypal_payment_error: isA.string().optional(),
   brand: isA.string().optional(),
+  paypal_payment_error: isA.string().optional(),
+  billing_agreement_id: isA.string().optional(),
```

Comment:
Can this also be `null`, e.g. if the billing agreement is invalid? I see it on the `Customer` object in fxa-payments-server/src/store/types.tsx  as typed `string | null`.

Reply:
Great catch, I was under the impression that `optional` took care of the null case here, but I see above for `billing_name` we are doing:

```
billing_name: isA
    .alternatives(isA.string(), isA.any().allow(null))
    .optional(),
```
That should work here as well.